internal/distro/rhel8/azure.go

```diff
@@ -98,6 +99,24 @@ func azureEapRhuiImgType() imageType {
 	it.name = "azure-eap-rhui"
 	it.nameAliases = nil // make sure we don't inherit aliases from the base image type
 	it.workload = eapWorkload()
+
+	// shell env vars for EAP
+	wildflyPath := "/opt/rh/eap7/root/usr/share/wildfly"
```

Comment:
I wonder if we should name the image type azure-eap7-rhui because the code is apparently somewhat tied to the EAP version...

Reply:
If we're going to continue building 7 when 8 comes around, then yes.  I don't know what EAP release/versioning looks like.

Reply:
I think that we can be on the safe side here and use `eap7` in the image name since the beginning. I can't think of any downside of doing so and it can potentially save us some headache in the future (similarly to the mess with distros `rhel-8` vs. `rhel-84`).